python/ray/data/_internal/progress_bar.py

```diff
@@ -113,6 +120,13 @@ def update(self, i: int) -> None:
                 self._bar.total = self._progress
             self._bar.update(i)
 
+    def update_total(self, total: int, estimate: bool = False) -> None:
```

Comment:
will leave this up to others to discuss, i think we don't need to explicitly show label newly updated total with a `~` "approximate" prefix, since we always provide the best estimate anyway (and it may confuse users).

Reply:
+1

I agree that it's useful to denote which numbers are estimations, but given that we already estimate our progress without labels, I'm not sure if we should do this in this PR.

Reply:
Sounds good!